hail/src/main/scala/is/hail/expr/ir/LowerMatrixIR.scala

```diff
@@ -288,6 +288,20 @@ object LowerMatrixIR {
               irRange(0, 'global(lengths)(colIdx), 1).map(Symbol(genUID()) ~> 'row(entriesField)(colIdx)))))
         .mapGlobals('global.dropFields(lengths))
 
+    case MatrixAggregateRowsByKey(child, entryExpr, rowExpr) =>
+      lower(child)
+        .aggregateByKey(
+          aggLet(va = 'row) {
+            rowExpr.insertFields(entriesField -> irRange(0, 'global (colsField).len)
```

Comment:
rowExpr is not used in here at all!

Reply:
`thing.insertFields(bindings...)` generates a 

```


```